samples/getOccurrence.js

```diff
@@ -0,0 +1,25 @@
+// [START containeranalysis_get_occurrence]
+// Retrieves and prints a specified Occurrence from the server
+const getOccurrence = async(
+    projectId = 'your-project-id', // Your GCP Project ID
+    occurrenceId = 'my-occurrence' // Your Occurrence name
```

Comment:
In Ruby, I used this as the comment text to be clear that it's not a user-generated id like the note id:

> "The API-generated identifier associated with the occurrence"